README.md

```diff
@@ -241,9 +241,9 @@ For actual use, provide a permanent key and cert in the manifest definition or e
 
 ## JSON schemas
 
-* [Schema for the Manifest Definition](schemas/manifest-definition.json)
+* [Schema for the Manifest Definition](https://github.com/contentauth/c2patool/blob/fix-doc-build/schemas/manifest-definition.json)
```

Comment:
These should be pointing to `.../blob/main/...` not `.../blob/fix-doc-build/...`

Reply:
Repeat for all 4 such links in this file.

Reply:
Doh!  Dumb mistake, thanks for catching!  Fixed them.